include/tmp/file

```diff
@@ -99,15 +99,27 @@ public:
                    std::error_code& ec);
 
   /// Creates a unique temporary copy from the given path
-  /// @param path      A path to make a temporary copy from
-  /// @param label     A label to attach to the temporary file path
-  /// @param extension An extension of the temporary file path
+  /// @param[in] path  A path to make a temporary copy from
+  /// @param[in] label A label to attach to the temporary file path
   /// @returns The new temporary file
   /// @throws std::filesystem::filesystem_error if given path is not a file
   /// @throws std::invalid_argument             if arguments are ill-formatted
   static file copy(const std::filesystem::path& path,
```

Comment:
And only now I realize that there is no way to specify whether it is binary or text